secator/runners/_base.py

```
@@ -508,9 +528,72 @@ def _finalize(self):
+			'run_options': dict(self.run_opts) if hasattr(self, 'run_opts') else {},
+			'reports_folder': str(self.reports_folder),
+			'tasks': tasks,
+		}
+
+		resume_path = f'{self.reports_folder}/resume.cfg'
+		try:
+			os.makedirs(self.reports_folder, exist_ok=True)
+			with open(resume_path, 'w') as f:
+				json.dump(data, f, indent=2)
+			self.debug(f'resume.cfg written to {resume_path}', sub='end')
+		except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)